Amasty_Storelocator/templates/pages/location_view.phtml

```diff
@@ -0,0 +1,305 @@
+<?php
+/**
+ * @author Amasty Team
+ * @copyright Copyright (c) 2020 Amasty (https://www.amasty.com)
+ * @package Amasty_Storelocator
+ */
+?>
+<?php
+/** @var \Amasty\Storelocator\Model\Location $location */
+/** @var \Amasty\Storelocator\Block\View\Location $block */
+$location = $block->getCurrentLocation();
+$location->setTemplatesHtml();
+
+$galleryImages = $block->getLocationGallery();
+
+$adressString = $location->getCity() . ', ' . $location->getAddress() . ', ' . $location->getZip();
+?>
+<div class="amlocator-location-main">
+    <div class="amlocator-location-container">
+        <div class="amlocator-column -left">
+            <div
+                class="
+                    amlocator-block
+                    -route
+                    amlocator-route-creator
+                "
+                data-amlocator-js="route-creator"
+            >
+                <div class="amlocator-modes">
+                    <input
+                        class="amlocator-mode"
+                        id="driving-mode"
+                        name="travel-mode"
+                        data-amlocator-js="travel-mode"
+                        type="radio"
+                        value="DRIVING"
+                        checked
+                    />
+                    <label
+                        class="amlocator-label -driving"
+                        for="driving-mode"
+                    ></label>
```

Comment:
Not really because this is what these fields looks like:
![Screenshot 2021-05-13 at 18 50 26](https://user-images.githubusercontent.com/32640873/118158463-163b4880-b41c-11eb-9191-dc36a2f42193.png)

Reply:
hmmm, could you please change colors from blue to some gray? only colors to make it nicer, icons should stay

Reply:
👍